src/OrchardCore.Modules/OrchardCore.Setup/Startup.cs

```diff
@@ -21,8 +22,8 @@ public Startup(IShellConfiguration shellConfiguration)
         {
             var configurationSection = shellConfiguration.GetSection("OrchardCore.Setup");
 
-            _defaultCulture = configurationSection["DefaultCulture"];
-            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<string[]>();
+            _defaultCulture = configurationSection["DefaultCulture"] ?? "";
+            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<List<string>>().ToArray();
```

Comment:
Again I'm not sure why we need to call `ToArray()` while it was an array before you convert it to `List <string>`

Reply:
If you set the supported culture like this with an empty string for InvariantCulture : 

```
    "OrchardCore.Setup": {
      "DefaultCulture": "en",
      "SupportedCultures": [ "", "en", "fr" ]
    }
```

And you do this : 

`_supportedCultures = configurationSection.GetSection("SupportedCultures").Get<List<string>>()` 

The array will contain a value that is converted to "null". 
{ null, en, fr }

You can't pass a null value to SupportedCulture as it will fail with an exception.

`_supportedCultures = configurationSection.GetSection("SupportedCultures").Get<List<string>>()` Will remove the string.empty value from the configuration and return only the 2 values that are not null.